app/components/primer/experimental/action_bar.pcss

```diff
@@ -0,0 +1,32 @@
+/* CSS for ActionBar  */
+
+.ActionBar {
+  display: flex;
+  align-items: center;
+  justify-content: end;
+  position: relative;
+}
+
+/* Divider */
+
+.ActionBar-divider {
+  margin: 0 var(--primer-control-medium-gap, 8px);
+  height: calc(var(--primer-control-medium-size, 32px) / 2);
+  border-left: var(--primer-borderWidth-thin, 1px) solid var(--color-border-subtle);
+}
+
+/* Increase spacing so touch targets don't overlap */
```

Comment:
Added this back to make sure there is enough spacing between items so that the touch targets don't overlap.

![Screen Shot 2022-08-31 at 14 08 40](https://user-images.githubusercontent.com/378023/187599121-f92c0289-66c5-4166-89e7-9b436a861460.png)